src/components/dashboard/SendLinkSummary.js

```diff
@@ -113,11 +113,9 @@ const SendLinkSummary = ({ screenProps }: AmountProps) => {
       let txHash
 
       // Generate link deposit
-      const generateLinkResponse = goodWallet.generateLink(
-        amount,
-        reason,
-        ({ paymentLink, code }) => (hash: string) => {
-          log.debug({ hash })
+      const generateLinkResponse = goodWallet.generateLink(amount, reason, {
+        onTransactionHash: ({ hash, paymentLink, code }) => {
```

Comment:
no need for special ontxhash
take paymetlink and code from generatelinkresponse

Reply:
done